### PR TITLE
Some fix to be able to upgrade from 2.2

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py [hunk 1]
```diff
@@ -75,6 +75,8 @@ def main():
     if options.nondocker:
         print("- nondocker")
     print("- git_remote=" + options.git_remote)
+    if options.use_makefile:
+        print("- use_makefile")
     print("- makefile=" + options.makefile)
```
Comment (ger-benjamin) [inline geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py]: looks a little bit strange, can you verify ?

What's the difference between "makefile" and "use_makefile" ? (also for the argument parser below (line 112))

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py]: The makefile is the make file used by c2cupgrade to lunch the build tasks, use_makefile is true if we lunch c2cupgrade true the makefile, it's used only tho display the commends used to run the different steps.

Comment (ger-benjamin) [reply geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py]: Ok, thanks

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py]: any idea?

Comment (ger-benjamin) [reply geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py]: Not really :sweat_smile:

**Pull request merged**